### PR TITLE
feat: log diff in interfaces check

### PR DESCRIPTION
**Pull request by maurelian**
**Description**

If the contracts interface check fails, this will print the diff to help with debugging.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 {
   "editorconfig.generateAuto": false,
-  "files.trimTrailingWhitespace": true
+  "files.trimTrailingWhitespace": true,
```
Comment (AmadiMichael) [inline .vscode/settings.json]: Are these changes intended?

Comment (maurelian) [reply .vscode/settings.json]: haha no.

Comment (maurelian) [reply .vscode/settings.json]: removed

**Review by smartcontracts — APPROVED** [review]
